tests/std/tests/VSO_0000000_regex_use/test.cpp

```diff
@@ -547,6 +547,7 @@ void test_VSO_226914_word_boundaries() {
 }
 
 void test_GH_993_regex_character_class_case_insensitive_search() {
+    // regex::icase is not handled correctly for some input.

```

Comment:
I think it is unnecessary repetition of function name and comment, and since the comment is clearer, suggest changing function name.

Reply:
Nevermind, other functions are named similar way

Reply:
even if they are, I'd prefer to standardize; are you willing to do that?

Reply:
I renamed some tests which I managed to find.
I searched: `grep -i test_gh */*/*/*/*.cpp` and  I renamed the found tests to the form: `test_gh_xxx`